src/Microsoft.AspNetCore.Razor.Language/Syntax/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -0,0 +1,1434 @@
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpMetaCode(CSharpMetaCodeSyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpCodeLiteral(CSharpCodeLiteralSyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpCodeBlock(CSharpCodeBlockSyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpStatement(CSharpStatement node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpStatementBody(CSharpStatementBodySyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpExpression(CSharpExpression node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpExpressionBody(CSharpExpressionBodySyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpDirective(CSharpDirectiveSyntax node)
+    {
+      return DefaultVisit(node);
+    }
+
+    public virtual TResult VisitCSharpDirectiveBody(CSharpDirectiveBodySyntax node)
+    {
+      return DefaultVisit(node);
+    }
+  }
+
+
+  internal partial class SyntaxVisitor
+  {
+    public virtual void VisitRazorCommentBlock(RazorCommentBlockSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitHtmlText(HtmlTextSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpTransition(CSharpTransitionSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpMetaCode(CSharpMetaCodeSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpCodeLiteral(CSharpCodeLiteralSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpCodeBlock(CSharpCodeBlockSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpStatement(CSharpStatement node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpStatementBody(CSharpStatementBodySyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpExpression(CSharpExpression node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpExpressionBody(CSharpExpressionBodySyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpDirective(CSharpDirectiveSyntax node)
+    {
+      DefaultVisit(node);
+    }
+
+    public virtual void VisitCSharpDirectiveBody(CSharpDirectiveBodySyntax node)
+    {
+      DefaultVisit(node);
+    }
+  }
+
+  internal static partial class SyntaxFactory
+  {
+    public static RazorCommentBlockSyntax RazorCommentBlock(SyntaxToken startCommentTransition, SyntaxToken startCommentStar, SyntaxToken comment, SyntaxToken endCommentStar, SyntaxToken endCommentTransition)
+    {
+#if DEBUG
```

Comment:
We don't have the same requirements as Roslyn. Having these checks here could help us prevent issues. I'd remove the `#if DEBUG` bits from the generated code when they involve argument checking.

Reply:
Sure. Will do